lib/auth0/api/authentication_endpoints.rb

```diff
@@ -7,7 +7,27 @@ module AuthenticationEndpoints
       UP_AUTH = 'Username-Password-Authentication'.freeze
       JWT_BEARER = 'urn:ietf:params:oauth:grant-type:jwt-bearer'.freeze
 
+      # Request an API access token using a Client Credentials grant
+      # @see https://auth0.com/docs/api-auth/tutorials/client-credentials
+      # @param audience [string] API audience to use
+      # @return [json] Returns the API token
+      # def api_token(audience = "https://#{@domain}/api/v2/")
+      def api_token(
+        client_id: @client_id,
+        client_secret: @client_secret,
+        audience: "https://#{@domain}/api/v2/"
```

Comment:
Wrapping this below in a `URI(audience).to_s` like you have in other gems is a good way to get some extra runtime guarantees, but is optional.

Reply:
The `audience` param could be anything, doesn't have to be a URI:

![screenshot 2018-09-20 09 48 28](https://user-images.githubusercontent.com/855223/45833808-76ed4400-bcba-11e8-8194-ad3121006e7e.png)